tests/framework/CodeInterfaceTests/abce_Custom_Sampler.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0"?>
+<Simulation verbosity="debug">
+  <RunInfo>
+    <JobName>abceGCI</JobName>
+    <Sequence>sweep</Sequence>
+    <WorkingDir>abcesetting</WorkingDir>
+    <batchSize>1</batchSize>
+  </RunInfo>
+
+  <Steps>
+    <MultiRun name="sweep">
+      <!-- <Input class="Files" type="">abce_runner.sh</Input> -->
```

Comment:
Old commented out lines should usually be removed unless there is a reason to keep them.